src/main/java/nl/tudelft/contextproject/gui/DirectorLiveController.java

```diff
@@ -87,14 +104,18 @@ private void initializeViews() {
         smallView.setImage(current);
     }
 
+    /**
+     * Initializes the methods.
```

Comment:
I would suggest a change in the javadoc here as it is not too acuurate in my opinion.